airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -72,23 +74,27 @@ def get_conn(self):
             num_retries=self.num_retries
         )
 
-    def get_service(self):
+    def get_service(self) -> Any:
         """
         Returns a BigQuery service object.
         """
         http_authorized = self._authorize()
         return build(
             'bigquery', 'v2', http=http_authorized, cache_discovery=False)
 
-    def insert_rows(self, table, rows, target_fields=None, commit_every=1000):
+    def insert_rows(
+        self, table: Any, rows: Any, target_fields: Any = None, commit_every: Any = 1000, replace: Any = False
```

Comment:
`Any` is not the best option. If you want proper types check service docs (REST API or Python SDK) or ask @mik-laj if he's refactoring this method.

Reply:
It's not implemented method, so it's not cotains valid types. This is part of the public interface from another class - `DbApiHook`, but this hook don't need this method.

Reply:
Hmmm I got it, but... those `Any` does not looks good to me 😿

Reply:
The method not implemented does not look good either. 😿

Reply:
What to do you think about putting `*args, **kwargs` there?

Reply:
The current version is more accurate. The current list of arguments may be useful to someone in the future.

Reply:
This is some information. Now it is of little use, but still information that we do not need to delete. This is not an inaccuracy.

Reply:
It looks ugly for me as well, but question is should I assign specific types despite there is no usage of them or it can stay 'ugly' by annotated?

Reply:
Or maybe this method can stay not annotated at all in this case. WDYT @mik-laj @nuclearpinguin ?

Reply:
No annotation propagates to other code fragments. Users see functions without annotations and then create theirs without annotations.

Reply:
In case of NotImplemented I am ok with `Any`. It would be nice to add some information how it could be implemented but I it's not necessary for me.